driverless/src/main/java/com/upgrad/hirewheels/controller/AuthenticationController.java

```diff
@@ -1,29 +1,42 @@
 package com.upgrad.hirewheels.controller;
 
```

Comment:
There is some issue with the implementation of Spring Security. We have added both the access token and refresh token  in the users table. At the same time we are maintaining the following data structures: 
1. List of tokens
2. User-Access Token Map
3. Access Token - Refresh Token Map
4. User - Refresh Token Map

Now, there are two questions:
1. We are not storing the tokens in DB even though we have created columns to store in the Users table. What is the logic behind this??
2. What is the use case of storing these tokens in different data structures? The rationale behind this is,
The owner of the JWT can easliy be verified using the signature and payload of the JWT. Please help me understand the logic here.

Reply:
I think that was done in haste by Sandeep... You are right, we should be storing at one place, either DB or inMemory ... I will go for the db choice!